Model/Article.php

```diff
@@ -0,0 +1,455 @@
+            $this->addFragment($fragment);
+        }
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFragments()
+    {
+        return $this->fragments;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setPublicationEndsAt(\DateTime $publicationEndsAt = null)
+    {
+        $this->publicationEndsAt = $publicationEndsAt;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPublicationEndsAt()
+    {
+        return $this->publicationEndsAt;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setPublicationStartsAt(\DateTime $publicationStartsAt = null)
+    {
+        $this->publicationStartsAt = $publicationStartsAt;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPublicationStartsAt()
+    {
+        return $this->publicationStartsAt;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setStatus($status)
+    {
+        $this->status = $status;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStatus()
+    {
+        return $this->status;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setSubtitle($subtitle)
+    {
+        $this->subtitle = $subtitle;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSubtitle()
+    {
+        return $this->subtitle;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setTags(Collection $tags = null)
+    {
+        $this->tags = $tags;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTags()
+    {
+        return $this->tags;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setTitle($title)
+    {
+        $this->title = $title;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUpdatedAt(\DateTime $updatedAt)
+    {
+        $this->updatedAt = $updatedAt;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getUpdatedAt()
+    {
+        return $this->updatedAt;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setValidatedAt(\DateTime $validatedAt = null)
+    {
+        $this->validatedAt = $validatedAt;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getValidatedAt()
+    {
+        return $this->validatedAt;
+    }
+
+    /**
+     * Validation.
+     *
+     * @param ClassMetadata $metadata
+     */
+    public static function loadValidatorMetadata(ClassMetadata $metadata)
+    {
+        $metadata->addConstraint(new Assert\Callback(array(__NAMESPACE__.'\\Article', 'validatorPublicationEnds')));
+    }
+
+    /**
+     * INTERNAL.
```

Comment:
Why? Could this be private or final protected?

BTW, you should use the `@internal` tag instead and add a reason if possible.